web/package.json

```diff
@@ -16,7 +16,11 @@
     "react": "^19.0.0",
     "react-dom": "^19.0.0",
     "react-icons": "^5.5.0",
-    "react-router": "^7.5.3"
+
+    "react-router": "^7.5.3",
+    "react-router-dom": "^7.6.2",
```

Comment:
we dont need both `react-router` and `react-router-dom` only `react-router` should stay